Tests/SentryProfilerTests/SentryAppLaunchProfilingTests.m

```diff
@@ -0,0 +1,178 @@
+#import "SentryLaunchProfiling+Tests.h"
+#import "SentryOptions+HybridSDKs.h"
+#import "SentryOptions+Private.h"
+#import "SentryProfilingConditionals.h"
+#import "SentrySDK+Tests.h"
+#import <XCTest/XCTest.h>
+
+#if SENTRY_TARGET_PROFILING_SUPPORTED
+
+@interface SentryAppLaunchProfilingTests : XCTestCase
+@end
+
+@implementation SentryAppLaunchProfilingTests
+
+#    define SENTRY_OPTION(name, value)                                                             \
+        NSStringFromSelector(@selector(name))                                                      \
+            : value
+
+- (void)testDefaultOptionsDoNotEnableLaunchProfiling
+{
+    XCTAssertFalse(shouldProfileNextLaunch([self defaultOptionsWithOverrides:nil]).shouldProfile,
```

Comment:
`m`: Nimble works for ObjC as well. I didn't try it yet, but it works great. Up to you.